addons/mass_mailing/tests/test_mailing_ab_testing.py

```diff
@@ -166,3 +166,11 @@ def test_mailing_ab_testing_minimum_participants(self):
             ab_testing.action_send_mail()
         self.assertEqual(ab_testing.state, 'done')
         self.assertEqual(len(self._mails), 1)
+
+    def test_mailing_ab_testing_duplicate_date(self):
+        """ Test that "Send final on" date value should be copied in new mass_mailing """
+        self.ab_testing_mailing_1.write({
+            'ab_testing_schedule_datetime': datetime.now() + timedelta(days=10),
+        })
+        new_mailing = self.ab_testing_mailing_1.copy()
```

Comment:
Would maybe instantiate a Form tool and hit 'action_duplicate' as it would match the functional usage of a/b testing.

Reply:
Looking at the code, actually not sure it adds much in the end, sorry :) note that you can normally give directly a record, like ``ab_testing_mail_1 = Form(self.ab_testing_mailing_1)`